packages/experimental/kai/src/containers/deprecated/TaskList.tsx

```diff
@@ -103,35 +103,33 @@ export const TaskItem: FC<{
   onDelete?: (task: Task) => void;
   onSave?: (task: Task) => void;
   slots?: EditableListItemSlots;
-}> = withReactor(
-  ({ task, onDelete, onSave, slots }) => {
-    useReactorContext({
-      onChange: () => {
-        onSave?.(task);
-      }
-    });
-
-    return (
-      <EditableListItem
-        id={task.id}
-        completed={task.completed}
-        title={task.title}
-        slots={{
-          ...slots,
-          listItem: {
-            selectableCheckbox: {
-              className:
-                'radix-state-checked:bg-white radix-state-unchecked:bg-white radix-state-checked:border radix-state-unchecked:border border-primary-600 text-primary-600'
-            }
+}> = observer(({ task, onDelete, onSave, slots }) => {
+  // TODO(wittjosiah): Remove?
+  // useReactorContext({
```

Comment:
@dmaretskyi after you removed this, this component didn't build. How should this behaviour be replicated?

Reply:
Use subscription would work here. But perhaps there’s another way to rewrite this code to not need the hook?